addons/repository.xbmc.org/addon.xml

```diff
@@ -8,10 +8,10 @@
   </requires>
 	<extension point="xbmc.addon.repository">
 		<info>http://mirrors.kodi.tv/addons/leia/addons.xml.gz</info>
-		<checksum>http://mirrors.kodi.tv/addons/leia/addons.xml.gz.md5</checksum>
+		<checksum>http://mirrors.kodi.tv/addons/leia/addons.xml.gz.sha256</checksum>
```

Comment:
can you change this to http://mirrors.kodi.tv/addons/leia/addons.xml.gz?sha256
which is the actual url mirrorbits uses. Saves 1 rewrite on the webserver.

Reply:
> Can we use the url from <info> and append ?sha256

We can, but that would break 3rd-party repos. Do you have a strong reason to change that?